releasing.md

```diff
@@ -120,6 +125,9 @@ ks param set --env=${ENV} workflows prow_env  \
 ks apply ${ENV} -c workflows
 ```
 
+Create a PR to update [jupyterhub_spawner.py](https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/jupyterhub_spawner.py#L15) 
+to point to the newly built Jupyter notebook images.
```

Comment:
also update all.jsonnet default value of tfJobImage?

Reply:
Done.